java/src/org/certificatetransparency/ctlog/serialization/Deserializer.java

```diff
@@ -114,12 +115,12 @@ public static ParsedLogEntry parseLogEntry(InputStream merkleTreeLeaf, InputStre
 
     if (entryType == Ct.LogEntryType.X509_ENTRY) {
       Ct.X509ChainEntry x509EntryChain = parseX509ChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getX509());
+          treeLeaf.timestampedEntry.signedEntry.x509);
       logEntry.x509Entry = x509EntryChain;
     } else if (entryType == Ct.LogEntryType.PRECERT_ENTRY) {
       Ct.PrecertChainEntry preCertChain = parsePrecertChainEntry(extraData,
-          treeLeaf.timestampedEntry.signedEntry.getPrecert());
-       logEntry.precertEntry = preCertChain;
+          treeLeaf.timestampedEntry.signedEntry.preCert);
+      logEntry.precertEntry = preCertChain;
```

Comment:
Am I the only one to find this "get a thing into a local, assign local to member variable of same name, never use local ever again" pattern kind of odd?